packages/react-composites/src/composites/common/PeoplePaneContent.tsx

```diff
@@ -66,6 +66,11 @@ export const PeoplePaneContent = (props: {
     [adapter]
   );
 
+  // dismiss drawers when participant leaves
+  adapter.on('participantsLeft', () => {
```

Comment:
We shouldn't be subscribing to the adapter events in our components. or composite. this should come from a selector if we are to do something when the participants leave.